db/schema.rb

```diff
@@ -10,11 +10,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190224103954) do
```

Comment:
Is there a reason for defining two versions? Shouldn't all prior versions are assumed to be run